docs/learn-phragmen.md

```diff
@@ -41,9 +41,102 @@ operating their nodes.
+2. An initial load of 0 is set for each ballot.
+3. The candidate who wins the next available seat is the one where the ballots of their supporters would have the _least average (mean) cost_ if that candidate wins.
+4. The _n_ ballots that approved that winning candidate get _1/n_ added to their load.
+5. The load of all ballots that supported the winner of this round are averaged out so that they are equal.
+6. If there are any more seats, go back to step 3.  Otherwise, the selection ends.
+
+Let's walk through an example with four candidates vying for three seats, and five voters.
+
+
+```
+Open Seats: 3
+
+Candidates:   A B C D  L0
+-------------------------
+Voter V1:       X      0
+Voter V2:         X X  0
+Voter V3:       X   X  0
+Voter V4:     X X      0
+Voter V5:       X X X  0
+```
+
+In this example, we can see that voter `V1` approves only of candidate `B`, voter `V2` approves of candidates `C` and `D`, etc.  Voters can approve any number of candidates between 1 and `number_of_candidates - 1`.  An initial "load" of `0` is set for each ballot (`L0` = load after round `0`, i.e., the "round" before the first round).  We shall see shortly how this load is updated and used to select candidates.
+
+We will now run through an iterative algorithm, with each iteration corresponding to one "seat".  Since there are three seats, we will walk through three rounds.
+
+For the first round, the winner is simply going to be the candidate with the most votes.  Since all loads are equal, the lowest average load will be the candidate with the highest n, since `1/n` will get smaller as `n` increases.  For this first example round, for instance, candidates `A` had only one ballot vote for them.  Thus, the average load for candidate A is `1/1`, or 1.  Candidate C has two ballots approving of them, so the average load is `1/2`.  Candidate B has the lowest average load, at `1/4` and they get the first seat.  Ballots loads are now averaged out, although for the first iteration, this will have not have any effect.
+
+```
+Filled seats: 1 (B)
+Open Seats: 2
+
+Candidates:   A B C D  L0 L1
+-----------------------------
+Voter V1:       X      0  1/4
+Voter V2:         X X  0  0
+Voter V3:       X   X  0  1/4
+Voter V4:     X X      0  1/4
+Voter V5:       X X X  0  1/4
+```
+
+We are now down to candidates `A`, `C`, and `D` for two open seats.  There is only one voter (`V4`) for `A`, with load `1/4`.   `C` has two voters, `V2` and `V5`, with loads of `0` and `1/4`.  `D` has three voters approving of them, `V2`, `V3`, and `V5`, with loads of `0`, `1/4`, and `1/4`, respectively.
+
+If Candidate `A` wins, the average load would `(1/4 + 1/1) / 1`, or `5/4`.  If candidate `C` wins, the average load would be `(0 + 1/2) + (1/4 + 1/2) / 2`, or `5/8`.  If candidate `D` wins, the average load would be `(0 + 1/3) + (1/4 + 1/3) + (1/4 + 1/3) / 3`, or `1/2`.  Since `1/2` is the lowest average load, candidate D wins the second round.
```

Comment:
```suggestion
If Candidate `A` wins, the average load would be `(1/4 + 1/1) / 1`, or `5/4`.  If candidate `C` wins, the average load would be `(0 + 1/2) + (1/4 + 1/2) / 2`, or `5/8`.  If candidate `D` wins, the average load would be `(0 + 1/3) + (1/4 + 1/3) + (1/4 + 1/3) / 3`, or `1/2`.  Since `1/2` is the lowest average load, candidate D wins the second round.
```